rel-eng/Makefile

```diff
@@ -1 +0,0 @@
-Makefile.git
```

Comment:
Also this is referenced in 28 other Makefiles

grep -rE "rel-eng\/Makefile$" .

Reply:
Those are not used in spec files: I checked them